### ISSUE TITLE
informed consent identification code

### ISSUE THREAD

**Issue by fjmanion** [ICO-ontology/ICO#2]
We have a class "informed consent identification code" that is a data item. There is no annotation on this code to indicate what it represents. Candidates for the annotation would logically include:
1) the IRB number associated with the approval of the informed consent instrument
2) a unique (system generated?) code or sequence number associated with the process of successfully executing a legally valid informed consent

I suspect the intent is (2), but this needs an authoritative definition from an accepted definitional source.

**Comment by mbrochhausen** [comment]
I agree that this needs clarification. It is not clear what the term is intended to represent.
